driver-sync/src/test/functional/com/mongodb/client/unified/RunOnRequirementsMatcher.java

```diff
@@ -89,6 +93,19 @@ public static boolean runOnRequirementsMet(final BsonArray runOnRequirements, fi
         return false;
     }
 
+    private static boolean serverlessMatches(final String serverlessRequirement) {
```

Comment:
I propose reusing `JsonTestServerVersionChecker.serverlessMatches` here. Note that the same has already been done for `JsonTestServerVersionChecker.topologyMatches`.

Reply:
Agreed.  Done.